includes/Generator/Plugins/OpenGraph.php

```diff
@@ -180,7 +180,7 @@ protected function addTitleMeta(): void {
 			$this->titlePropertyName, Html::element(
 				'meta', [
 					self::$htmlElementPropertyKey => $this->titlePropertyName,
-					self::$htmlElementContentKey => $this->outputPage->getHTMLTitle(),
+					self::$htmlElementContentKey => $this->outputPage->getTitle()->getPrefixedText(),
```

Comment:
```suggestion
					self::$htmlElementContentKey => $this->outputPage->getTitle()->getPrefixedText(),
```

`getTitle()`  can return `null` which would cause a nullpointer error, do something like:
```php
		$title = $this->outputPage->getHTMLTitle();
		if ( $this->outputPage->getTitle() !== null ) {
			$title = $this->outputPage->getTitle()->getPrefixedText() ?? $title;
		}
```